src/components/K8s.vue

```diff
@@ -51,8 +63,56 @@ export default {
   computed: {
     cannotReset: function() {
       return (this.state !== K8s.State.STARTED && this.state !== K8s.State.READY);
+    },
+
+    memoryValueIsValid: function() {
+      let value = this.settings.kubernetes.memoryInGB;
+      let numericValue;
+      if (typeof(value) !== "number") {
+        numericValue = parseFloat(value, 10);
+        if (isNaN(numericValue)) {
+          return false;
+        }
+      }
+      if (numericValue < 2 ||
+          (numericValue > this.availMemoryInGB && this.availMemoryInGB)) {
+        return false;
+      }
+      return true;
+    },
+
+    numCPUsValueIsValid: function() {
+      let value = this.settings.kubernetes.numberCPUs;
+      let numericValue;
+      if (typeof (value) !== "number") {
+        numericValue = parseInt(value, 10);
+        if (isNaN(numericValue)) {
+          return false;
+        }
+      } else {
+        numericValue = value;
+      }
+      if (numericValue < 2 ||
```

Comment:
Why is giving the machine 1 CPU bad? Especially since the UI lets me give it 1 CPU… kind of; it doesn't stick.

Reply:
The code has changed, and `MIN_CPUS` is set to 1.